db/utils.js

```diff
@@ -145,6 +145,8 @@ if (conf.readReplicas) {
   seq = new Sequelize(env.dbUrl, opts);
 }
 
+seq.Promise = Sequelize.Promise; // seq v5
+
```

Comment:
Yes, because it is being passed on to the all the models. And I changed it below as well because its more clear that it is called on a class rather than an an instance of the class.